<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerBord</class>
 <widget class="QWidget" name="ManagerBord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>611</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_book">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <attribute name="title">
     <string>书籍</string>
    </attribute>
    <widget class="QListView" name="lw_books">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>601</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_addBook">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>＋</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_delBook">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>－</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_editBook">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>馆内可借阅书籍：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_reader">
    <attribute name="title">
     <string>读者</string>
    </attribute>
    <widget class="QListView" name="lw_readers">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>601</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_changePassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>修改管理员密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_logout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>更改用户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
